src/components/Notifications/NotificationFeed/NotificationFeed.js

```diff
@@ -350,7 +351,23 @@ class NotificationFeed extends React.Component {
     return newNotifications
   }
 
-  onDeleteAllClick = () => {}
+  onDeleteAllClick = () => {
+    const newList = []
+    let pagination = this.state.pagination ?? {}
+    if (pagination) {
+      pagination = {
+        ...pagination,
+        total_items: pagination.total_items > 0 ? pagination.total_items - 1 : 0,
+      }
+    }
+
+    this.setState({
+      notificationList: newList,
+      pagination,
```

Comment:
If pagination is 0 will any notification be deleted?

Reply:
The value of pagination won't affect the deleteAllNotifications() endpoint, but the UI/UX.
Pagination can only be an empty object rather than 0.
If pagination is {}, notifications will get deleted as well.